src/main/java/org/opensearch/ad/model/ADTask.java

```diff
@@ -26,6 +25,8 @@
 import org.opensearch.core.xcontent.XContentBuilder;
 import org.opensearch.core.xcontent.XContentParser;
 import org.opensearch.timeseries.annotation.Generated;
+import org.opensearch.timeseries.model.DateRange;
```

Comment:
DetectionDateRange was in the same package/internal class?

Reply:
Ok, got it - you now need to reference this externally?

Reply:
yes, since I changed 

src/main/java/org/opensearch/ad/model/DetectionDateRange.java
 
to

./src/main/java/org/opensearch/timeseries/model/DateRange.java